lang/en/alt.php

```diff
@@ -5,7 +5,7 @@
 return [
     "prev" => "previous",
     "next" => "next",
-    "shadow" => "blue shadow",
+    "shadow-sm" => "blue shadow-sm",
```

Comment:
Did you really want to change that?

Reply:
After upgrade Tailwind from v3 to v4 now I'm looking for some strange stuff - thanks :blush:

Reply:
Eventually, it is unnecessary key, so I deleted it.